modules/migration/review.go

```diff
@@ -36,16 +41,18 @@ func (r *Review) GetExternalID() int64 { return r.ReviewerID }
 
 // ReviewComment represents a review comment
 type ReviewComment struct {
-	ID        int64
-	InReplyTo int64 `yaml:"in_reply_to"`
-	Content   string
-	TreePath  string `yaml:"tree_path"`
-	DiffHunk  string `yaml:"diff_hunk"`
-	Position  int
-	Line      int
-	CommitID  string `yaml:"commit_id"`
-	PosterID  int64  `yaml:"poster_id"`
-	Reactions []*Reaction
-	CreatedAt time.Time `yaml:"created_at"`
-	UpdatedAt time.Time `yaml:"updated_at"`
+	ID          int64
+	InReplyTo   int64 `yaml:"in_reply_to"`
+	Content     string
+	TreePath    string `yaml:"tree_path"`
+	DiffHunk    string `yaml:"diff_hunk"`
+	Position    int
+	Line        int
+	CommitID    string `yaml:"commit_id"`
+	PosterID    int64  `yaml:"poster_id"`
+	PosterName  string
```

Comment:
Because this field is being added in this PR, it's hard to go make a own PR for this.

But this struct would need to add 2 functions:

```go
// GetExternalName ExternalUserMigrated interface
func (r *ReviewComment) GetExternalName() string { return r.PosterName }

// ExternalID ExternalUserMigrated interface
func (r *ReviewComment) GetExternalID() int64 { return r.PosterID }
```

And in gitea_uploader.go at line 970 you would need to pass the comment instead of the review trough, this will fix that the author is the actual author instead of the one that made the review:

Before:
![image](https://user-images.githubusercontent.com/25481501/152678390-1121f0bc-8e5f-416f-a23b-ae26019698ab.png)
After:
![image](https://user-images.githubusercontent.com/25481501/152678398-386c3625-cb7c-4268-8d90-b7ccfa73dbbb.png)

Original on github: https://github.com/userstyles-world/userstyles.world/pull/57#discussion_r628955084

Reply:
Note: this is a bug that existed before this PR. It just so happens to come into light now.